tests/dummy/app/pods/docs/why-ember-table/template.md

```diff
@@ -0,0 +1,125 @@
+# Why Ember Table?
+
+## Tables are Hard
+
+They've been with us, for better or for worse, since the early
+days of computing, helping us sort through mounds of data and generate TPS
+reports. Excel macros are practically Turing complete at this point, and just
+about every UX pattern imaginable has been explored and implemented in a table
+in some application somewhere.
+
+Building that functionality into your Single Page App is a long, tiresome
+process, and as the datasets scale it just gets harder. There are a lot of
+different off the shelf tables out there, like ag-grid and HandsOnTable, but
```

Comment:
not sure if we want to reference other tables.

Reply:
I can remove the references. Mainly wanted to highlight the differences, and why we're better for many use cases.

Reply:
My main concern is that I don't know if this could feel outdated each time someone else try to publish a new addon.